### PR TITLE
Update the release README

### PR DESCRIPTION
**Pull request by yphus**
## Documentation

New workflows are now available to release checkbox using a unique tag.

This is the documentation update corresponding to: 
- https://github.com/canonical/checkbox/pull/402
- https://github.com/canonical/checkbox/pull/408

### CODE REVIEW

File: tools/release/README.md [hunk 1]
```diff
@@ -1,4 +1,57 @@
-# Checkbox release process (Debian packages)
+# Checkbox release process
+
+> **_NOTE:_** Check the [Launchpad Builders status] before triggering a release.
+Sometimes, the builders will be down and this might prevent building the
+packages.
+
+## Bump the checkbox version
```
Comment (pieqq) [inline tools/release/README.md:7]: Checkbox should be uppercased to avoid confusion

File: tools/release/README.md [hunk 2]
```diff
@@ -1,4 +1,57 @@
...
+Run the [Bumpversion workflow] (the default increment is **minor**)
```
Comment (pieqq) [inline tools/release/README.md:8]: Missing a column at the end of the sentence, and probably a blank line between L8 and L9.

File: tools/release/README.md [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
+# Checkbox release process
+
+> **_NOTE:_** Check the [Launchpad Builders status] before triggering a release.
+Sometimes, the builders will be down and this might prevent building the
+packages.
+
+## Bump the checkbox version
+Run the [Bumpversion workflow] (the default increment is **minor**)  
+Steps in this workflow:
+- Run [Bumpversion] for all checkbox sub-projects, snaps and debian packages
```
Comment (pieqq) [inline tools/release/README.md:10]: `checkbox`, `snaps` and `debian` should be uppercased.

File: tools/release/README.md [hunk 4]
```diff
@@ -1,4 +1,57 @@
...
+- [checkbox snap packages] *(built and uploaded to their respective beta channels)*
+- [checkbox core snap packages] *(built and uploaded to their respective beta channels)*
+
+In addition to the above workflows, a draft release is created on Github with
+an auto-generated changelog.
+
+Check the related Github Action logs to see if everything runs as expected:
+
+- snapcraft is not blocked during the snap build process. For example, in this 
+[build](https://github.com/canonical/checkbox/actions/runs/4371649401/jobs/7649877336), the i386 build was blocked on an error (`Chroot problem`) for hours
```
Comment (pieqq) [inline tools/release/README.md:42]: Not sure if you want to cap each line at 80 characters, in which case this one has to be adapted (you can put the link at the bottom of the document, for instance).

File: tools/release/README.md [hunk 5]
```diff
@@ -1,4 +1,57 @@
...
+- [checkbox deb packages] *(built and published to the testing PPA)*
+- [checkbox snap packages] *(built and uploaded to their respective beta channels)*
+- [checkbox core snap packages] *(built and uploaded to their respective beta channels)*
+
+In addition to the above workflows, a draft release is created on Github with
+an auto-generated changelog.
+
+Check the related Github Action logs to see if everything runs as expected:
+
+- snapcraft is not blocked during the snap build process. For example, in this
```
Comment (pieqq) [inline tools/release/README.md:41]: `snapcraft` should be uppercased.

File: tools/release/README.md [hunk 6]
```diff
@@ -1,4 +1,57 @@
...
+
+In addition to the above workflows, a draft release is created on Github with
+an auto-generated changelog.
+
+Check the related Github Action logs to see if everything runs as expected:
+
+- snapcraft is not blocked during the snap build process. For example, in this 
+[build](https://github.com/canonical/checkbox/actions/runs/4371649401/jobs/7649877336), the i386 build was blocked on an error (`Chroot problem`) for hours 
+before finally completing
+- the expected number of snaps are built. snapcraft
```
Comment (pieqq) [inline tools/release/README.md:44]: `snapcraft` should be uppercased.

File: tools/release/README.md [hunk 7]
```diff
@@ -1,4 +1,57 @@
...
+In addition to the above workflows, a draft release is created on Github with
+an auto-generated changelog.
+
+Check the related Github Action logs to see if everything runs as expected:
+
+- snapcraft is not blocked during the snap build process. For example, in this 
+[build](https://github.com/canonical/checkbox/actions/runs/4371649401/jobs/7649877336), the i386 build was blocked on an error (`Chroot problem`) for hours 
+before finally completing
+- the expected number of snaps are built. snapcraft 
+does not return 1 when only a few of the snaps are built, which leads to GHA
```
Comment (pieqq) [inline tools/release/README.md:45]: I would either write GitHub Actions, either use the `<abbr>` tag to explain what `GHA` means:

```
<abbr title="GitHub Actions">GHA</abbr>
```

**Review by pieqq — CHANGES_REQUESTED** [review]
Thanks for providing this!

I just have some nitpicks about the form (see inline) and also the fact that it's not clear if you try to cap each line at 80 chars or not, because some paragraphs seem to be following this while others don't.

But if you don't want to make the changes, I think it's OK to land as is.

**Comment by yphus** [comment]
@pieqq thx for the sugestions, the last commit should fix them all

**Review by pieqq — APPROVED** [review]
All good then, thanks!

**Pull request merged**